test-cases/longevity/longevity-large-partition-3h.yaml

```diff
@@ -40,3 +40,7 @@ nemesis_interval: 15
 user_prefix: 'longevity-large-partitions-3h'
 
 space_node_threshold: 644245094
+
+run_fullscan: '{"ks_cf": "scylla_bench.test", "interval": 10}' # 'ks.cf|random, interval(min)'
+
+run_full_partition_scan: '{"ks_cf": "scylla_bench.test", "interval": 2, "pk_name":"pk", "rows_count": 5000, "validate_data": "true"}' # 'ks.cf, interval(min), partition-key name, number-of-rows-per-partition, validate reversed query output'
```

Comment:
default now for `validate_data` should be `"false"`

Reply:
fixed